Cargo.lock

```diff
@@ -11506,3 +11622,8 @@ dependencies = [
  "libc",
  "pkg-config",
 ]
+
+[[patch.unused]]
+name = "getrandom"
+version = "0.2.9"
+source = "git+https://github.com/madsim-rs/getrandom.git?rev=8daf97e#8daf97e4142635fe28543b2db9022f5e2544bb5c"
```

Comment:
👀

Reply:
Yep, madsim also needs to update the dependency. I'll change to madsim source when @wangrunji0408 publishes the new version.

Reply:
published as madsim-aws-sdk-s3 v0.3.0

Reply:
Btw, getrandom has updated to v0.2.11 some days ago. Although there's nothing important according to its [changelog](https://github.com/rust-random/getrandom/blob/master/CHANGELOG.md). If we are going to bump its version, remember to bump the patch version as well:

```toml
getrandom = { git = "https://github.com/madsim-rs/getrandom.git", rev = "e79a7ae" }
```

Reply:
> If we are going to bump its version, remember to bump the patch version as well:

It was eventually forgotten.

Reply:
🤣